src/fprime_gds/executables/cli.py

```diff
@@ -501,10 +500,11 @@ def handle_arguments(cls, args, **kwargs):
 
         # Handle configuration arguments
         config = fprime_gds.common.utils.config_manager.ConfigManager()
-        if args.config is not None and os.path.isfile(args.config):
-            config.set_configs(args.config)
-        elif args.config is not None:
-            raise ValueError(f"Configuration file {args.config} not found")
+        if args.config is not None:
```

Comment:
In the python in mainline, we had issue with the if-depth of this kind of refactor.  This seems to be in the same category of changes.  Thoughts?

Reply:
I don't like nested statements either, but it seemed to me that these were more astute (code duplication and readability).